Video:
  FRAME_WIDTH: 640
  FRAME_HEIGHT: 480

SmartCar:
  k_val: [0.6, 0.3, 1.8, 0.9, 0.8, 0.4] # scale, center, dist
  scale_default: 0.5
  dist_default: 0.8
  by_dist: True
